# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../openvas-manager/openvas-manager-2.0.3-no-sysconfdir-suffix.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- openvas-manager-2.0.3/CMakeLists.txt.orig	2011-04-28 00:11:37.992000255 +0200
+++ openvas-manager-2.0.3/CMakeLists.txt	2011-04-28 00:12:40.920000254 +0200
@@ -150,7 +150,7 @@
 set (OPENVAS_STATE_DIR   "${LOCALSTATEDIR}/lib/openvas")
 set (OPENVAS_LOG_DIR     "${LOCALSTATEDIR}/log/openvas")
 set (OPENVAS_PID_DIR     "${LOCALSTATEDIR}/run")
-set (OPENVAS_SYSCONF_DIR "${SYSCONFDIR}/openvas")
+set (OPENVAS_SYSCONF_DIR "${SYSCONFDIR}")
 
 set (OPENVAS_LIB_INSTALL_DIR     "${LIBDIR}")
 set (OPENVAS_HEADER_INSTALL_DIR  "${INCLUDEDIR}")
